chat-widget/automation_tests/e2e/areas/general/chatButton.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { Browser, BrowserContext } from "playwright";
+import * as playwright from "playwright";
+import { TestSettings } from "../../../configuration/test-settings";
+import { BasePage } from "../../pages/base.page";
+import { CustomLiveChatWidgetConstants } from "e2e/utility/constants";
+
+describe("live chat button", () => {
+    let newBrowser: Browser;
+    let context: BrowserContext;
+    let page: BasePage;
+
+    beforeEach(async () => {
+        newBrowser = await playwright[TestSettings.Browsers as any].launch(
+            TestSettings.LaunchBrowserSettings
+        );
+        context = await newBrowser.newContext({
+            viewport: TestSettings.Viewport,
+        });
+    });
+
+    afterEach(async () => {
+        if (context) {
+            await context.close();
+        }
+        if (newBrowser) {
+            await newBrowser.close();
+        }
+    });
+
+    test("chat button visible", async () => {
+        page = new BasePage(await context.newPage());
+        await page.openLiveChatWidget();
+        const isLiveChatButtonVisible = await page.waitUntilLiveChatSelectorIsVisible(
```

Comment:
We can use an expect statement to test if the liveChatButton is visible:
expect(await page.Page.isVisible(CustomLiveChatWidgetConstants.LiveChatButtonId)).toBeTruthy();

Otherwise, this test will pass even if the button is not there

Reply:
TODO: this is just framework PR wanted to add. In next PR will add expect